airflow/providers/apache/hive/example_dags/example_twitter_dag.py

```diff
@@ -137,7 +106,7 @@ def transfertodb():
 
     for channel in to_channels:
 
-        file_name = "to_" + channel + "_" + yesterday.strftime("%Y-%m-%d") + ".csv"
+        file_name = "to_" + channel + "_" + dt + ".csv"
```

Comment:
Should we use f-string btw?

Reply:
Yeah we should. I have a habit of going a little deeper on changes that the defined scope sometimes so I was holding myself back. I'll make sure this gets updated.